src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameCollector.cs

```diff
@@ -8,11 +8,16 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using System.Security.Cryptography;
```

Comment:
Is it worth (or needed) to mix this into Blame collector when the parameters used on commandline are not under "blame"?

Reply:
The idea is that Blame can control every "postmortem" system used in a agnostic way...it simply check if some dump with the correct name are generated under that folder...it's unrelated to procdump it could work also on WER for instance.